identity/client/e2e/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "testing-library/prefer-screen-queries": "off",
+    "testing-library/no-await-sync-query": "off",
```

Comment:
There's also a Playwright eslint plugin that I think it's useful

Reply:
I don't see anything on Operate nor Tasklist's config for ESLint, do you know the name of it?